tests/pipeline/test_engine.py

```diff
@@ -271,6 +315,123 @@ def test_numeric_factor(self):
             DataFrame(expected_avg, index=dates, columns=self.assets),
         )
 
+    def test_rolling_and_nonrolling(self):
+        open_ = USEquityPricing.open
+        close = USEquityPricing.close
+        volume = USEquityPricing.volume
+
+        # Test for thirty days up to the second to last day that we think all
+        # the assets existed.  If we test the last day of our calendar, no
+        # assets will be in our output, because their end dates are all
+        dates_to_test = self.dates[-32:-2]
```

Comment:
My intended change was to merge in the MultiColumnLoaderTests, since I removed the MultiColumnLoader.

I couldn't really follow [the original intent] (https://github.com/quantopian/zipline/commit/26fd6fda8b095d052374fb27812dca3a6cad4869#diff-34c73f1261aca33c4c7c106482806900R509). What do you suggest?